cypress/support/commands.js

```diff
@@ -10,7 +10,30 @@
 //
 //
 // -- This is a parent command --
-// Cypress.Commands.add("login", (email, password) => { ... })
+Cypress.Commands.add("login", (email, password) => {
+
+  // it is ok for the email to be visible in the Command Log
+  expect(email, "email was set").to.be.a("string").and.not.be.empty;
+  // but the password value should not be shown
+  if (typeof password !== "string" || !password) {
+    throw new Error("Missing password value, set using CYPRESS_password=...");
+  }
+
+  cy.get('input[name="emaial"]')
+    .type(email)
+    .should("have.value", email);
```

Comment:
this test is redundant in my opinion, checks that you typed the value that you just typed - it will always pass.

Reply:
Good point. Done